.github/workflows/gems-bump-version.yml

```diff
@@ -0,0 +1,71 @@
+name: Gems - Bump Version
+on:
+  workflow_dispatch:
+    inputs:
+      version_type:
+        description: "Version Type?"
+        required: true
+        type: choice
+        options:
+          - minor
+          - patch
```

Comment:
`major` is omitted simply because the `bump-version.rb` script doesn't currently support it.

TBH, I think that's just fine for now, major version bumps have never actually happened yet, so we can wait to add support until we actually need it.